### PR TITLE
chore: install ptl in startup hooks

### PR DESCRIPTION
**Pull request by mpkouznetsov**
Build and document determined after removing PyTorch Lightning and Torch Metrics from environment images.

All examples using PyTorch Lightning have been modified to include installing PTL in their startup hooks.

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](../docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/release-notes/4377-install-ptl-in-startup-hooks.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+:orphan:
```
Comment (mpkouznetsov) [inline docs/release-notes/4377-install-ptl-in-startup-hooks.txt:1]: @gh-determined-ai non-trivial change #1

File: docs/training/apis-howto/api-pytorch-lightning-ug.rst [hunk 2]
```diff
@@ -105,6 +105,12 @@ In addition, we also patched some ``LightningModule`` methods to make porting yo
    Determined will automatically log the metrics you return from ``training_step`` and
    ``validation_step`` to Tensorboard.
 
+.. note::
```
Comment (mpkouznetsov) [inline docs/training/apis-howto/api-pytorch-lightning-ug.rst:108]: @gh-determined-ai non-trivial change #2 . Please check my wording.

File: docs/release-notes/4377-install-ptl-in-startup-hooks.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+
+**Security Fixes** **Breaking Changes**
+
+-  PyTorch Lightning is no longer a part of determined environments. When needed, it should be
```
Comment (gh-determined-ai) [inline docs/release-notes/4377-install-ptl-in-startup-hooks.txt:5]: Capitalize "Determined"

File: docs/training/apis-howto/api-pytorch-lightning-ug.rst [hunk 4]
```diff
@@ -105,6 +105,12 @@ In addition, we also patched some ``LightningModule`` methods to make porting yo
...
+
+   Determined environment images no longer contain PyTorch Lightning. To use PyTorch Lightning, you
```
Comment (gh-determined-ai) [inline docs/training/apis-howto/api-pytorch-lightning-ug.rst:110]: Suggestion ...
Determined environment images no longer include PyTorch Lightning. To use PyTorch Lightning, add a line similar to the following in the ``startup-hooks.sh`` script:

File: docs/training/apis-howto/api-pytorch-lightning-ug.rst [hunk 5]
```diff
@@ -105,6 +105,12 @@ In addition, we also patched some ``LightningModule`` methods to make porting yo
...
+   have to install it by adding a line like the following to ``startup-hooks.sh``: ``pip install
```
Comment (gh-determined-ai) [inline docs/training/apis-howto/api-pytorch-lightning-ug.rst:111]: It would be better to have the line to be added in a codeblock instead of inline.

File: docs/training/apis-howto/api-pytorch-lightning-ug.rst [hunk 6]
```diff
@@ -105,6 +105,12 @@ In addition, we also patched some ``LightningModule`` methods to make porting yo
...
+   pytorch_lightning==1.5.10 torchmetrics==0.5.1``
+
 To learn about this API, you can start by reading the trial definitions from the following examples:
```
Comment (gh-determined-ai) [inline docs/training/apis-howto/api-pytorch-lightning-ug.rst:114]: Not your change but, ideally, remove "you can".

**Review by gh-determined-ai — CHANGES_REQUESTED** [review]

**Review by gh-determined-ai — APPROVED** [review]
lgtm
